src/satellite_consumer/run.py

```diff
@@ -36,6 +36,24 @@
 except PackageNotFoundError:
     __version__ = "v?"
 
+#Sentry Integration
+import sentry_sdk
+
+sentry_sdk.init(
+    dsn=os.getenv("SENTRY_DSN"), 
+    environment=os.getenv("ENVIRONMENT", "local"), 
+    traces_sample_rate=1,
+)
+
+sentry_sdk.set_tag("app_name", "satellite_consumer")
+
+# __version__ might not be available in dev/test environments
+try:
+    sentry_sdk.set_tag("version", __version__)
+except Exception:
+    sentry_sdk.set_tag("version", "unknown")
```

Comment:
I dont thnk you need this try and except, as __version__ is always defined, see lines 34 to 37